db/migrate/20150513082710_change_status_on_rents.rb

```diff
@@ -0,0 +1,20 @@
+class ChangeStatusOnRents < ActiveRecord::Migration
+  def up
+    change_column_default(:rents, :status, nil)
+    # Should this be done in raw SQL or smth instead?
+    # d.wessman 2015-05-13
+    Rent.find_each do |rent|
+      if rent.status == 'Ej bestämd' || rent.status == 'Ej bestÃƒÂ¤md'
```

Comment:
The point is that some rents have this fault status because of a previous encoding error.
But I definitely agree with doing it in the database :) Would properly need some help with pure SQL so it will be right.